contracts/cryptoblades.sol

```diff
@@ -388,8 +386,8 @@ contract CryptoBlades is Initializable, AccessControlUpgradeable {
     }
 
     function mintCharacter() public doesNotHaveMoreThanMaxCharacters oncePerBlock(msg.sender) requestPayFromPlayer(mintCharacterFee) {
-        require(characters.balanceOf(msg.sender) < characterLimit,
-            string(abi.encodePacked("You can only have ",characterLimit," characters!")));
+        require(characters.balanceOf(msg.sender) < characters.characterLimit,
+            string(abi.encodePacked("You can only have ",characters.characterLimit," characters!")));
```

Comment:
I believe state variables of other contracts cannot be accessed like this. They need to be treated like getter functions. Also requires the state variable in question to be `public`.

```suggestion
        require(characters.balanceOf(msg.sender) < characters.characterLimit(),
            string(abi.encodePacked("You can only have ",characters.characterLimit()," characters!")));
```